app/components/Button/index.jsx

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames'
+import React from 'react'
```

Comment:
Good point. It is tempting to import the specific names under PropTypes as well (or bind them with something like `{ string, bool, arrayOf } = PropTypes` to avoid repeating PropTypes as well, but that does open up the possibility of accidental name collisions there that would only be reliably caught by static type checking.